packages/angular/ng-package.json

```diff
@@ -14,7 +14,6 @@
     "@phenomnomnominal/tsquery",
     "@typescript-eslint/",
     "chalk",
-    "chokidar",
```

Comment:
Do you know why this was here? Should we also remove `ignore` from here?

Reply:
the angular package used to have its own file-server that had chokidar and ignore. I can remove it too so that its cleaner.